### ISSUE TITLE
Cannot specify requirejs dependencies from a CDN

### ISSUE THREAD

**Issue by ccapndave** [dbashford/mimosa#141]
Firstly I should say that whilst writing this bug out I found the solution, so really the bug is just that it would be good to have this in the documentation somewhere!

---

Sometimes I want to specify dependencies from a CDN in order to take advantage of local caching:

``` coffeescript
require.config {
  urlArgs: "b=#{(new Date()).getTime()}"
  paths:
    jquery: 'http://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min'

require ['jquery'], ($) ->
  console.log "Here I am!"
```

This works fine until I want to optimize with r.js at which point I use the instructions at http://requirejs.org/docs/optimization.html#empty to add an entry to overrides in mimosa-`config.coffee`:

``` coffeescript
overrides:
        paths:
            jquery: "empty:"
```

However, at runtime I get an error stating `undefined missing jquery`.  It turns out that this is because almond doesn't support loading requirements over the network, so the solution is to tell mimosa to use the normal requirejs client:

``` coffeescript
overrides:
        name: "vendor/require"
        paths:
            jquery: "empty:"
```
